components/html-block/README.md

```diff
@@ -0,0 +1,23 @@
+# HTML Block
+
+## d2l-html-block
+
+The `d2l-html-block` element is a web component for displaying user-authored HTML. It includes styles for headings, lists, anchors and other elements.  In addition, it provides integration with MathJax for rendering MathML.
+
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/html-block/html-block.js';
+</script>
+
+<d2l-html-block html="..."></d2l-html-block>
```

Comment:
Thoughts about an attribute vs. using a nested `<template>`? I think we're already doing that in the LMS to be able to defer rendering... although the attribute also accomplishes that.

The ergonomics of a template seem nicer if you're just crafting a manual block for testing/demo (no HTML encoding needed), but in practice folks will likely just be binding to the attribute via Lit so it won't matter.

Reply:
I was thinking about this too and I was sorta on the fence about it, which is the main reason I started this PR as a draft.  😏   It would indeed make demo snippets a little nicer.  

Are you thinking `d2l-html-block` would require the content be provide in a `template`?  If it's in a `template`, the component can clone it and stamp it into local shadowDOM within the scope of the styles (`h1, h2, etc`), whereas if the HTML is rendered to the lightDOM, it would be outside the scope of those selectors, making applying those styles more difficult to apply, since the consumer context may be any arbitrary web component's shadowDOM.  

Rendering the HTML in the shadowDOM has the opposite consequence though - the consumer can't target elements in the user-authored HTML - something that I think is done in one or two places in the LMS to override default paragraph margins. Hoping we can find another sol'n to that, and such a use case should not exist.

Reply:
Yeah I was thinking it would need to be in a `<template>`, which would allow it to be lazy loaded. Otherwise I think the browser is going to start downloading and executing scripts/images/videos right away before our web component even boots up.